### PR TITLE
fix resize for GPT, be more verbose for logging

### PR DESCRIPTION
**Pull request by ogra1**
- we are missing valuable info in the resize logs, so add -p everywhere
- resize2fs fails with the new quoting when $resizeopts is empty (like it was in the GPT case), 
  this issue is fixed as a side effect of adding -p everywhere.

### CODE REVIEW

File: initramfs/scripts/local-premount/resize [hunk 1]
```diff
@@ -103,11 +103,12 @@ if [ "$min_free_space" -lt "$free_space" ]; then
         gpt)
             # do_gpt needs the device name
             do_gpt "$device" >>$LOGFILE 2>&1
+            resizeopts="-p"
```
Comment (zyga) [inline initramfs/scripts/local-premount/resize]: Can you please add a flag or test ensuring that `resizeopts` is never empty so that this bug doesn't come back with some refactoring later on. Please include a comment describing why that is needed.
